app/src/main/java/com/sun_asterisk/moviedb_50/screen/home/SliderViewPagerAdapter.kt

```diff
@@ -0,0 +1,71 @@
+package com.sun_asterisk.moviedb_50.screen.home
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import androidx.viewpager.widget.PagerAdapter
+import com.sun_asterisk.moviedb_50.R
+import com.sun_asterisk.moviedb_50.data.model.Movie
+import com.sun_asterisk.moviedb_50.data.source.remote.GetImageAsyncTask
+import com.sun_asterisk.moviedb_50.utils.Constant
+import com.sun_asterisk.moviedb_50.utils.OnClickListener
+import com.sun_asterisk.moviedb_50.utils.OnFetchImageListener
+import kotlinx.android.synthetic.main.item_slide.view.*
+
+class SliderViewPagerAdapter(
+    private val context: Context,
+    private val list: List<Movie>
+) : PagerAdapter() {
+    private lateinit var slideItemClickListener: OnClickListener<Movie>
+
+    override fun instantiateItem(container: ViewGroup, position: Int): Any {
+        val inflater =
+            context.getSystemService(Context.LAYOUT_INFLATER_SERVICE) as LayoutInflater
```

Comment:
nếu mà chỉ truyền context qua adapter vì đoạn này thì có thể dùng 
```suggestion
            container.context.getSystemService(Context.LAYOUT_INFLATER_SERVICE) as LayoutInflater
```